### PR TITLE
Fixes #1208: OngoingTopicList - Tablet (Portrait) (Lowfi)

### LINKED ISSUE #1208: OngoingTopicList - Tablet (Portrait) (Lowfi)
Mocks: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/93603b3c-47b6-437b-9542-fcfeb43beeaf/P-Topics-in-Progress  

Implement low-fi UI for **OngoingTopicList** tablet portrait mode

**Target PR date**: 30 July 2020
**Target completion date**: 3 August 2020

### PR DESCRIPTION
**Pull request by MohamedMedhat1998**
## Explanation

- Implemented low-fi UI for **OngoingTopicList** for portrait mode - tablet
- Fixes #1208 
## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Review by rt4914 — CHANGES_REQUESTED** [review]
LGTM, we will need to write test cases for span count check. similar to `ProfileChooserSpanTest` for this file and all the rest of the earlier files too.

**Comment by MohamedMedhat1998** [comment]
> 
> 
> LGTM, we will need to write test cases for span count check. similar to `ProfileChooserSpanTest` for this file and all the rest of the earlier files too.

Done!  
@rt4914  PTAL

File: app/src/main/res/layout-land/ongoing_topic_list_fragment.xml [hunk 1]
```diff
@@ -57,6 +57,7 @@
         android:paddingTop="28dp"
         android:paddingBottom="88dp"
         android:scrollbars="none"
+        android:tag="TAG_ONGOING_RECYCLER_VIEW"
```
Comment (rt4914) [inline app/src/main/res/layout-land/ongoing_topic_list_fragment.xml]: Set this in `strings.xml` and use it whereever you want. If in future someone changes the tag here, even the test cases will start failing, so `strings.xml` approach will work in a better way.

File: app/src/main/res/layout-sw600dp-port/ongoing_topic_list_fragment.xml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:clipToPadding="false"
+        android:overScrollMode="never"
+        android:paddingStart="20dp"
+        android:paddingTop="20dp"
+        android:paddingEnd="20dp"
+        android:paddingBottom="144dp"
+        android:scrollbars="none"
+        android:tag="TAG_ONGOING_RECYCLER_VIEW"
```
Comment (rt4914) [inline app/src/main/res/layout-sw600dp-port/ongoing_topic_list_fragment.xml]: Ditto

File: app/src/main/res/layout/ongoing_topic_list_fragment.xml [hunk 3]
```diff
@@ -56,6 +56,7 @@
         android:paddingEnd="20dp"
         android:paddingBottom="144dp"
         android:scrollbars="none"
+        android:tag="TAG_ONGOING_RECYCLER_VIEW"
```
Comment (rt4914) [inline app/src/main/res/layout/ongoing_topic_list_fragment.xml]: Ditto

File: app/src/test/java/org/oppia/app/testing/OngoingTopicListSpanTest.kt [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+      }
+    }
+  }
+
+  @Test
+  @Config(qualifiers = "sw600dp-land")
+  fun testOngoingTopicListFragmentRecyclerView_tablet_land_hasCorrectSpanCount() {
+    launch(OngoingTopicListActivity::class.java).use { scenario ->
+      scenario.onActivity { activity ->
+        assertThat(getOngoingRecyclerViewGridLayoutManager(activity).spanCount).isEqualTo(4)
```
Comment (rt4914) [inline app/src/test/java/org/oppia/app/testing/OngoingTopicListSpanTest.kt:93]: Here you have mentioned `4` but the `sw600dp-land` dimens file does not contain `ongoing_topics_span_count`

**Review by rt4914 — CHANGES_REQUESTED** [review]
PTAL

File: app/src/test/java/org/oppia/app/testing/OngoingTopicListSpanTest.kt (hunk 4 above)
Comment (MohamedMedhat1998) [reply app/src/test/java/org/oppia/app/testing/OngoingTopicListSpanTest.kt:93]: It will be added in the PR related to low-fi land

**Comment by BenHenning** [comment]
@MohamedMedhat1998 please update this with the latest develop.

**Comment by MohamedMedhat1998** [comment]
> 
> 
> @MohamedMedhat1998 please update this with the latest develop.

Done

File: app/src/test/java/org/oppia/app/testing/OngoingTopicListSpanTest.kt (hunk 4 above)
Comment (MohamedMedhat1998) [reply app/src/test/java/org/oppia/app/testing/OngoingTopicListSpanTest.kt:93]: added

File: app/src/main/res/layout/ongoing_topic_list_fragment.xml (hunk 3 above)
Comment (MohamedMedhat1998) [reply app/src/main/res/layout/ongoing_topic_list_fragment.xml]: done

File: app/src/main/res/layout-sw600dp-port/ongoing_topic_list_fragment.xml (hunk 2 above)
Comment (MohamedMedhat1998) [reply app/src/main/res/layout-sw600dp-port/ongoing_topic_list_fragment.xml]: done

File: app/src/main/res/layout-land/ongoing_topic_list_fragment.xml (hunk 1 above)
Comment (MohamedMedhat1998) [reply app/src/main/res/layout-land/ongoing_topic_list_fragment.xml]: done

File: app/src/test/java/org/oppia/app/testing/OngoingTopicListSpanTest.kt [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+  ): OngoingTopicListFragment {
+    return activity
+      .supportFragmentManager
+      .findFragmentByTag(
+        OngoingTopicListFragment.TAG_ONGOING_TOPIC_LIST_FRAGMENT
+      ) as OngoingTopicListFragment
+  }
+
+  @Test
+  fun testOngoingTopicListFragmentRecyclerView_hasCorrectSpanCount() {
```
Comment (rt4914) [inline app/src/test/java/org/oppia/app/testing/OngoingTopicListSpanTest.kt]: Rename to `testOngoingTopicList_checkRecyclerView_hasCorrectSpanCount`

Ditto for other tests.

File: app/src/main/java/org/oppia/app/ongoingtopiclist/OngoingTopicListFragment.kt [hunk 6]
```diff
@@ -12,6 +12,7 @@ import javax.inject.Inject
 class OngoingTopicListFragment : InjectableFragment() {
 
   companion object {
+    internal const val TAG_ONGOING_TOPIC_LIST_FRAGMENT = "TAG_ONGOING_TOPIC_LIST_FRAGMENT"
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/ongoingtopiclist/OngoingTopicListFragment.kt]: Rename to `ONGOING_TOPIC_LIST_FRAGMENT_TAG`

**Review by rt4914 — CHANGES_REQUESTED** [review]
Nit changes suggested.

File: app/src/test/java/org/oppia/app/testing/OngoingTopicListSpanTest.kt (hunk 5 above)
Comment (MohamedMedhat1998) [reply app/src/test/java/org/oppia/app/testing/OngoingTopicListSpanTest.kt]: done

File: app/src/main/java/org/oppia/app/ongoingtopiclist/OngoingTopicListFragment.kt (hunk 6 above)
Comment (MohamedMedhat1998) [reply app/src/main/java/org/oppia/app/ongoingtopiclist/OngoingTopicListFragment.kt]: done

**Review by rt4914 — APPROVED** [review]
LGTM

**Pull request merged**